src/parser/shared/modules/StatTracker.js

```diff
@@ -455,12 +455,29 @@ class StatTracker extends Analyzer {
     this.statBuffs[buffId] = stats;
   }
 
+  /**
+   * Akin to StatTracker.add(), but for buff providing stat multipliers rather than flat rating increases.
+   * @param buffId ID of the stat buff.
+   * @param stats Object with stats (intellect, mastery, etc.) and their respective stat modifier (E.g. if a buff increases a stat by 8%, the modifier is 1.08).
+   */
+  addStatMultiplierBuff(buffId, stats) {
+    if (!buffId || !stats) {
+      throw new Error(`StatTracker.addStatMultiplierBuff() called with invalid buffId ${buffId} or stats`);
+    }
+    if (typeof buffId === 'object') {
+      buffId = buffId.id;
+    }
+    if (this.statMultiplierBuffs[buffId]) {
+      throw new Error(`Stat buff with ID ${buffId} already exists`);
+    }
+    this.statMultiplierBuffs[buffId] = stats;
```

Comment:
This is incorrect. Seraphim is not a stat multiplier. For example: Battle Shout increases your agility/strength by 5%---by multiplying by 1.05. 50 mainstat becomes 52.5. Seraphim increases your secondaries by *adding* 8% to each: 50% becomes 58%, not 54%.

Reply:
However, once again I'll say: we don't need to support the non-Haste buffs right this second. We can revisit them down the line if you don't want to add support for this kind of buff right now. The haste portion is the most important one and is already covered by the `Haste` module.

Reply:
That makes sense and clears up some of the confusion I was having. I'll adjust the analyzer to support only the haste buff for the time being.